plugins/services/src/js/pages/task-details/TaskFileViewer.js

```diff
@@ -240,4 +229,23 @@ TaskFileViewer.propTypes = {
   task: PropTypes.object
 };
 
+TaskFileViewer.prototype.getNewRoute = function getNewRoute(
```

Comment:
I'd suggest either extract the function to a util or writing a snapshot test. Let's please not sacrifice conventions to accommodate testing.

Reply:
It was pretty hard to get an instance set up correctly, that's why I wanted to export a function.
Exporting also didn't work because of the `module.exports` thing so I went the easier route. I will investigate how to do this

Reply:
What's the `module.exports` thing? We use es6 modules in some newer places in the codebase if that'd help you.

Reply:
We could also pair to see how we could mock the setup

Reply:
The test broke and I think I correlated some UI break which was a coincidence and associated that there is a special reason not to use imports here. Is there a reason we don't use imports all over the place?

Reply:
We have some cyclic dependencies I think that was the reason we don't use es6 imports everywhere

Reply:
I think mocking makes it worse in that area